setup.py

```diff
@@ -41,23 +41,22 @@
         "Operating System :: OS Independent",
         "Programming Language :: Python",
         "Programming Language :: Python :: 3",
-        "Programming Language :: Python :: 3.7",
         "Programming Language :: Python :: 3.8",
         "Programming Language :: Python :: 3.9",
         "Programming Language :: Python :: 3.10",
+        "Programming Language :: Python :: 3.11",
+        "Programming Language :: Python :: 3.12",
         "Framework :: Django",
-        "Framework :: Django :: 2.2",
-        "Framework :: Django :: 3.0",
-        "Framework :: Django :: 3.1",
         "Framework :: Django :: 3.2",
+        "Framework :: Django :: 4.2",
+        "Framework :: Django :: 5.0",
         "Framework :: Wagtail",
-        "Framework :: Wagtail :: 2",
-        "Framework :: Wagtail :: 3",
         "Framework :: Wagtail :: 4",
+        "Framework :: Wagtail :: 5",
     ],
     install_requires=[
-        "Django>=3.2,<4.2",
-        "Wagtail>=2.15,<5.0",
+        "Django>=3.2,<5.1",
+        "Wagtail>=4.1",
```

Comment:
I am so tempted to say >= 5.2 🙊

Reply:
I mean, February 1st... I see your point. I guess it wouldn't make a difference since people upgrading from a 4.0 version could land on the current version, which supports that. I'm happy to go with that route

Reply:
Done